docker-load-execute.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -e
+
+start_api() {
+    echo 'Starting API'
+    export DB_URL=postgresql
+    export DB_PORT=5432
+    export DB_NAME=intermediary-test
+    export DB_USER=intermediary
+    export DB_PASS=changeIT!
+    export DB_SSL=require
+    ./gradlew shadowJar
+    docker compose up --build -d
+    docker compose logs router --no-color > docker-load-test-logs.txt
+    export API_PID="${!}"
+    echo "API starting at PID ${API_PID}"
+}
+
+start_database() {
+    echo 'Starting database'
+    docker compose -f docker-compose.postgres-test.yml up -d
+    sleep 2
+    echo "Database started"
+}
+migrate_database() {
+    echo 'Migrating database'
+    liquibase update --changelog-file ./etor/databaseMigrations/root.yml --url jdbc:postgresql://localhost:5434/intermediary-test --username intermediary --password 'changeIT!' --label-filter '!azure'
+    echo "Database migrated"
+}
+
+wait_for_api() {
+    attempt_counter=0
+    max_attempts=36
+
+    until curl --output /dev/null --silent --head --fail http://localhost:8080/health; do
+        if [ "${attempt_counter}" -eq "${max_attempts}" ];then
+            echo 'Done waiting for API to respond'
+            exit 1
+        fi
+        attempt_counter=$(($attempt_counter+1))
+        echo 'Waiting for API to respond'
+        sleep 5
+    done
+
+    echo 'API is responding'
+}
+
+warm_up_api() {
+    echo 'Warming up API...'
+    pem=$(pwd)/mock_credentials/organization-trusted-intermediary-private-key-local.pem
+    token=$(jwt encode --exp='+5min' --jti $(uuidgen) --alg RS256  --no-iat -S@${pem})
```

Comment:
You could read the JWT from the mock_credentials .jwt file to remove the dependency with the `jwt` CLI

Reply:
Part of the reason for this is to also warm up the Auth endpoint. I found that individually calling the endpoints helps those endpoints perform faster.

Reply:
What I mean is that you could replace the `jwt` CLI command, for something like this: `cat mock_credentials/trusted-intermediary-valid-token.jwt`

Reply:
Ah I see, for some reason I thought the `mock_credentials/trusted-intermediary-valid-token.jwt` was the latter step in auth but that makes sense. I'll change this asap.